### PR TITLE
add kustomize

### PR DESCRIPTION
**Pull request by developer-guy**
Signed-off-by: Batuhan Apaydın <batuhan.apaydin@trendyol.com>

### CODE REVIEW

File: kustomize.yaml [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+  contents:
+    packages:
+      - bash
+      - curl
+      - ca-certificates-bundle
+      - busybox
+pipeline:
+  - runs: |
+      curl -sSLO https://raw.githubusercontent.com/kubernetes-sigs/kustomize/master/hack/install_kustomize.sh
+      bash install_kustomize.sh ${{package.version}}
```
Comment (dlorenc) [inline kustomize.yaml:25]: Can we build from source? It's unclear what this script will do.

Comment (developer-guy) [reply kustomize.yaml]: done @dlorenc

**Comment by dlorenc** [comment]
Needs a rebase but otherwise LGTM!

**Review by rawlingsj — APPROVED** [review]

**Comment by developer-guy** [comment]
kindly ping @kaniini @dlorenc ☝️

**Review by amouat — APPROVED** [review]

**Pull request merged**